cafebabel/core/helpers.py

```diff
@@ -59,3 +59,17 @@ def file_exceeds(file_, size):
     blob = file_.read()
     file_.seek(0)
     return len(blob) > size
+
+
+def current_language():
+    lang_domain = current_app.config.get('DOMAIN_LANGUAGES')
+    domain_lang = {v: k for k, v in lang_domain.items()}
+    return domain_lang.get(request.host)
+
+
+def absolute(url):
+    if not url:
+        return ''
+    if url.startswith('/'):
+        url = url[1:]
+    return f'{request.url_root}{url}'
```

Comment:
I found this a bit hackish and prefer the `url_for` way. Not that neither that implementation nor the `url_for` deal with multiple domains like we will have to do (at least to redirect to the appropriate translation)

Reply:
I feel like there's a very little in common. `url_for` converts a func name into a url. `absolute` converts a relative url into an absolute.
So the url could be a result of `url_for` but is a different process.
How would you see this merged with `url_for`?

Reply:
`url_for` with `_external=True` parameter gives you an absolute URL (as I previously did).

Reply:
How would you suggest managing the `article.image_url` in that case?
Should we get back with a method that return the url with the `_external=True` option?

Reply:
We should have a dedicated view for `uploads` (vs. the default `static`) and use it from templates directly with the `_external` parameter, WDYT?